tdms/src/electric_field.cpp

```diff
@@ -0,0 +1,16 @@
+#include "complex"
+#include "globals.h"
+#include "field.h"
+
+using namespace std;
+
+
+void ElectricField::add_to_angular_norm(double f, int n, int Nt, SimulationParameters &params) {
+  angular_norm += phasor_norm(f, n, params.omega_an, params.dt, Nt);
+}
+
+complex<double> ElectricField::phasor_norm(double f, int n, double omega, double dt, int Nt){
+  return f
+         * exp( fmod(omega*((double) (n+1))*dt, 2*dcpi) * I)
```

Comment:
What's `I` set to (in the exponent)? It's not an input nor a class attribute?

Reply:
the imaginary unit defined in globals.h:

https://github.com/t-young31/ifdtd/blob/289fbdc50b7a441c5bce7c37d611bb6ba75c07c7/tdms/include/globals.h#L73